src/imgui/ImGuiSettings.cc

```diff
@@ -61,6 +61,21 @@ ImGuiSettings::~ImGuiSettings()
 	deinitListener();
 }
 
+void ImGuiSettings::initDefaultShortcuts()
+{
+	shortcuts[ShortcutIndex::GOTO_ADDRESS] = ImGuiMod_Ctrl | ImGuiKey_G; // Ctrl+G
+}
```

Comment:
imgui.ini saves all widget states so if we consider ImGui shortcuts as widget state, we should save them in imgui.ini, right? How do you imagine it interacting with settings.xml? Maybe just checking for collisions?

Reply:
From a pragmatic point of view:
* We have already an option to switch GUI layout, currently that's implemented via switching to a different .ini file. (As-if you copy it over the `imgui.ini`)
* I don't consider GUI shortcuts part of the GUI layout.  (I mean I'd find it annoying if switching GUI layout would also switch GUI shortcuts).
* The current bindings are stored in `settings.xml`. And shortcuts are closely related to bindings.

For these reason my idea was that the new shortcut settings belong in `settings.xml` rather than in `imgui.ini`. But this is only my initial idea, I admit I haven't given this much thought yet.

Reply:
If Shortcuts live inside ImGuiManager, how should I get access to Shortcuts from SettingsConfig? I could go through  
```
auto& manager = commandController.getReactor().getImGuiManager();
```
if I change commandController's reference to GlobalCommandController.

Reply:
In the prototype phase anything is fine.
Towards the final version I'd propose to create a new class `Shortcuts` dedicated for this.
We can then still decide what location is the most appropriate owner for the (single) instance of that class (e.g. it could be ImGuiManager, or it could be Reactor).